src/System.Collections.Immutable/src/System.Collections.Immutable.csproj

```diff
@@ -43,7 +43,7 @@
     <Compile Include="System\Collections\Generic\ISortKeyCollection.cs" />
     <Compile Include="System\Collections\Immutable\AllocFreeConcurrentStack.cs" />
     <Compile Include="System\Collections\Immutable\DictionaryEnumerator.cs" />
-    <Compile Include="System\Collections\Immutable\DisposableEnumeratorAdapter`2.cs" />
+    <Compile Include="System\Collections\Immutable\DisposableEnumeratorAdapter.Generic.cs" />
```

Comment:
@hughbe: please replace all ` and + characters with an _ underscore character, and remove other changes you've introduced. We'll consider that an adequate workaround till the nuget source server bugs are fixed.